ui/app/templates/vault/cluster/secrets/backend/sign.hbs

```diff
@@ -76,48 +76,41 @@
       <MessageError @model={{this.model}} />
       <NamespaceReminder @mode="sign" @noun="SSH key" />
       {{#if this.model.attrs}}
-        {{#each (take 1 this.model.attrs) as |attr|}}
```

Comment:
cleaned up and moved `valid_principals` outside of the more options toggle because _typically_ it will be required 
<img width="1711" alt="Screenshot 2024-09-23 at 5 26 34 PM" src="https://github.com/user-attachments/assets/df955e45-8abd-417e-a65d-bab272f70ace">